### PR TITLE
mercurial: update to 4.9

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

- bump version to 4.9

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.3 18D109
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by ra1nb0w** [comment]
Probably we can delete -devel considering that nobody maintain the port. @seanfarley what do you think?

File: devel/mercurial/Portfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ PortGroup           bitbucket 1.0
 name                mercurial
 # don't forget to update dependents for mercurial:
```
Comment (mojca) [inline devel/mercurial/Portfile]: Please read the comments :)

**Comment by mojca** [comment]
And yes, I agree that removing the development ports would probably be in place, I was just too lazy to do it with all the replacements in place etc.

File: devel/mercurial/Portfile (hunk 1 above)
Comment (ra1nb0w) [reply devel/mercurial/Portfile]: Yes, you are right. I did the updated port one month ago and I commit this during a clean without re-check. Sorry for the mistake. So, probably, I can close this and follow-up your PR.

**Pull request closed without merging**